js/apps/system/_admin/aardvark/APP/package.json

```diff
@@ -18,7 +18,7 @@
     "eslint-plugin-node": "^7.0.1",
     "eslint-plugin-promise": "^4.0.1",
     "eslint-plugin-standard": "^4.0.0",
-    "grunt": "^1.0.3",
+    "grunt": "^1.0.4",
```

Comment:
Also please do not add package.json.

Reply:
Should this file then be in `.gitignore`?

Reply:
No! But we should not update the package.json contents without further investigation.

Reply:
Raising the version here (for any package) could possibly break something.

Reply:
I learned "on the internet" that packages.json shouldn't be in `.gitignore`, so I abandoned that idea.